### ISSUE TITLE
Angular 6 webpack configuration

### ISSUE THREAD

**Issue by inunotaisho26** [angular/angular-cli#10670]
We will close this issue if you don't provide the needed information.
For feature requests, delete the form below and describe the requirements and use case.

I am unable to find the current webpack config file due to `eject` being disabled. The reason I need this info is because I build my personal angular apps without the cli. That being said the cli has been high influential  in my product infrastructure. 

Therefore, in order for me update to angular 6, I need to see the current webpack configuration. 

I have other request, but on thing at a time obviously. Want to keep a much knucklehead stuff out of the community as possible.

**Comment by vincentjames501** [comment]
+1

**Comment by inunotaisho26** [comment]
@filipesilva We need help with this issue.

**Comment by vincentjames501** [comment]
Ideally, and as many others have commented, there would just be a way to customize webpack w/o having to eject via some sort of hook system. It's frustrating having to eject and manage these things independently when all I want to add is a single webpack plugin.

**Comment by larssn** [comment]
@vincentjames501 This plugin https://github.com/Angular-RU/angular-cli-webpack tried to do just that. But it's utterly broken since the release of NG6.

I've been trying to fix it, but no luck so far, the CLI API has changed a great deal since the RC's it seems.

**Comment by bluefire2121** [comment]
Lazy loading fails with **Uncaught TypeError: Cannot read property 'call' of undefined**. It can't be fixed unless there's access to the webpack.config.js file

**Comment by usiyalla** [comment]
+1

**Comment by xane29** [comment]
+1

**Comment by larssn** [comment]
I made a proposal here https://github.com/angular/angular-cli/issues/1548#issuecomment-387136761

**Comment by envil** [comment]
Duplicated from this issue: https://github.com/angular/angular-cli/issues/10618

**Comment by prasannavenkateshrao** [comment]
I am facing the same issue with Angular 6 wherein I am not able to setup webpack config. I used CLI for setting up.  Eject is disabled and I have no clue on getting my webpack as I want to customize it. Any help is really appreciated.

**Comment by eBitzu** [comment]
I think you can do it manually if in angular.json yo do this changes:

under "architect": "build"  you change builder to "@angular-devkit/build-webpack:webpack" 

Let me know how it went.

Sources:
https://medium.com/@meltedspark/customizing-angular-cli-6-build-an-alternative-to-ng-eject-a48304cd3b21
https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/build_webpack

**Comment by kavi87** [comment]
It's annoying having to fork the cli everytime to allow merging a webpack config, I don't understand what the issue is really. "My config does not work because I added custom stuff in it". Well, that's your problem, not the cli one. No one is going to create an issue if they are responsible, or the issue can be closed immediately.

**Comment by meltedspark** [comment]
> Ideally, and as many others have commented, there would just be a way to customize webpack w/o having to eject via some sort of hook system. It's frustrating having to eject and manage these things independently when all I want to add is a single webpack plugin.

@vincentjames501 I agree with you 💯 %, never understood why it is so hard to merge configs instead of ejecting. I guess it's just getting low priority in comparison to other CLI features. 
Anyways, you can use [this](https://github.com/meltedspark/angular-cli-builders#custom-webpack-browser) extension to achieve this behavior 👍
